eventuate-tram-sagas-quarkus-orchestration/src/main/java/io/eventuate/tram/sagas/quarkus/orchestration/SagaManagerImplInitializer.java

```diff
@@ -16,7 +16,7 @@
   private Logger logger = LoggerFactory.getLogger(this.getClass());
 
   @Inject
-  Instance<SagaManager> sagaManagers;
+  Instance<SagaManager<?>> sagaManagers;
```

Comment:
Please write a test that is fixed by this change.

Reply:
will do